_data/sites.json

```
@@ -440,6 +440,15 @@
         "notes_it-IT": "L'unico modo per contattarli è tramite email, ma nessuno risponde."
     },
 
+    {
+        "name": "Modrinth",
+        "url": "https://modrinth.com/legal/privacy",
+        "difficulty": "hard",
+        "email": "gdpr@modrinth.com",
+        "notes": "Send an email requesting a copy of your data.",
+        "notes_it": "Invia una email richiedendo una copia dei tuoi dati."
```

The check is failing, because the field should be named `notes_it-IT`, instead of `notes_it`.

```suggestion
        "notes_it-IT": "Invia una email richiedendo una copia dei tuoi dati."
```